.github/workflows/gen-binaries.yml

```
@@ -0,0 +1,40 @@
+name: Generate embedded binaries
```

I'm not 100% sure but we might want to split the binary generation workflows because this repo will potentially contain multiple generated binaries.

If so we might want to rename this file to something like `gen-nodetime-binaries.yml` and change the workflow title to:

```suggestion
name: Generate nodetime embedded binaries
```